src/components/manners/implementation.ts

```diff
@@ -45,10 +23,35 @@ export type Implementation = {
      * Various file system hooks.
      */
     hooks: {
-      afterLoadExisting: (fs: FileSystem, account: FS.AssociatedIdentity, dataComponents: DataComponents) => Promise<void>
-      afterLoadNew: (fs: FileSystem, account: FS.AssociatedIdentity, dataComponents: DataComponents) => Promise<void>
-      beforeLoadExisting: (cid: CID, account: FS.AssociatedIdentity, dataComponents: DataComponents) => Promise<void>
-      beforeLoadNew: (account: FS.AssociatedIdentity, dataComponents: DataComponents) => Promise<void>
+      afterLoadExisting: (fs: FS, depot: Depot.Implementation) => Promise<void>
+      afterLoadNew: (fs: FS, depot: Depot.Implementation) => Promise<
+        null | {
+          path: Path.Distinctive<Path.Segments>
+          capsuleRef: PrivateReference
+        }
+      >
+      beforeLoadExisting: (cid: CID, depot: Depot.Implementation) => Promise<void>
+      beforeLoadNew: (depot: Depot.Implementation) => Promise<void>
+    }
+  }
+
+  /**
+   * Cabinet configuration.
+   *
+   * Responsible for storing UCANs and keys.
+   */
+  cabinet: {
+    hooks: {
+      inventoryChanged: (cabinetItems: CabinetCollection) => Promise<void>
     }
   }
-}
```

Comment:
Very cool. Can see this hook being useful in a bunch of places. 💯

Reply:
I just noticed I accidentally exposed the `cabinet` naming here. Should we rename this to `authority` and/or move it to the authority component? 🤔

Reply:
Removed the hook for now, added a note to add an event for this later when we figure out the details of the authority interface.

Reply:
Sounds good 👌